manim/mobject/geometry.py

```diff
@@ -336,43 +362,50 @@ def point_at_angle(self, angle):
         return self.point_from_proportion((angle - start_angle) / TAU)
 
 
+@attr.s(auto_attribs=True, eq=False)
 class Dot(Circle):
-    CONFIG = {
-        "radius": DEFAULT_DOT_RADIUS,
-        "stroke_width": 0,
-        "fill_opacity": 1.0,
-        "color": WHITE,
-    }
+    radius: float = DEFAULT_DOT_RADIUS
+    stroke_width: float = 0
+    fill_opacity: float = 1.0
+    color: tp.Union[str, Color] = WHITE
+    point: tp.Any = ORIGIN
 
-    def __init__(self, point=ORIGIN, **kwargs):
-        Circle.__init__(self, arc_center=point, **kwargs)
+    def __attrs_post_init__(self):
+        self.arc_center = self.point
+        Circle.__attrs_post_init__(self)
 
 
+@attr.s(auto_attribs=True, eq=False)
 class SmallDot(Dot):
-    CONFIG = {
-        "radius": DEFAULT_SMALL_DOT_RADIUS,
-    }
+    radius: float = DEFAULT_SMALL_DOT_RADIUS
 
+    def __attrs_post_init__(self):
+        Dot.__attrs_post_init__(self)
 
+
+@attr.s(auto_attribs=True, eq=False)
 class Ellipse(Circle):
-    CONFIG = {"width": 2, "height": 1}
+    width: float = 2.0
+    height: float = 1.0
 
-    def __init__(self, **kwargs):
-        Circle.__init__(self, **kwargs)
+    def __attrs_post_init__(self):
+        Circle.__attrs_post_init__(self)
         self.set_width(self.width, stretch=True)
         self.set_height(self.height, stretch=True)
 
 
+@attr.s(auto_attribs=True, eq=False)
 class AnnularSector(Arc):
-    CONFIG = {
-        "inner_radius": 1,
-        "outer_radius": 2,
-        "angle": TAU / 4,
-        "start_angle": 0,
-        "fill_opacity": 1,
-        "stroke_width": 0,
-        "color": WHITE,
-    }
+    inner_radius: float = 1.0
+    outer_radius: float = 2.0
+    angle: float = TAU / 4
+    start_angle: float = 0
```

Comment:
that or change to 0.0
```suggestion
    start_angle: tp.Union[int, float] = 0
```
```suggestion
    start_angle: float = 0.0
```